linux/gnome/src/main/cpp/GnomeThemeUtils.cpp

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
Generally there is some odd spacing with angle `<` brackets in this files.

Reply:
That was caused by running `autostyleApply`. Despite the directives in `cpp.eclipseformat.xml` about angle brackets and spaces, autostyle still tries to make spaces before and between the angle brackets. Are you able to replicate this on your end too?

Reply:
I’ll have to check. Maybe the eclipseformat needs to be adjusted. Weird though that it sometimes puts spaces around the brackets and sometimes not.

Reply:
Yeah. That's the other weird part. It doesn't seem to be consistent. From what I can see, it only wants spaces when the angle brackets are being used for a variable declaration whose type doesn't have its own type.

`std::vector<std::string> resources_dirs_paths` gets corrected but
`std::vector<std::map<std::string, std::string>> installed_resources` does not.

Usage inside a function also seems to not need spaces as `resource.insert(std::pair<std::string, std::string>("path", resource_dir->get_path()))` was not changed.